npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/pnpm_lockfile_updater.rb

```diff
@@ -66,8 +66,8 @@ def run_pnpm_updater
           end.join(" ")
 
           Helpers.run_pnpm_command(
-            "install #{dependency_updates} --lockfile-only --ignore-workspace-root-check",
-            fingerprint: "install <dependency_updates> --lockfile-only --ignore-workspace-root-check"
+            "install #{dependency_updates} --lockfile-only",
+            fingerprint: "install <dependency_updates> --lockfile-only"
```

Comment:
The failing tests suggest we should add this back

```suggestion
            "install #{dependency_updates} --lockfile-only --ignore-workspace-root-check",
            fingerprint: "install <dependency_updates> --lockfile-only --ignore-workspace-root-check"
```